packages/client/katello-host-tools/katello-host-tools.spec

```
@@ -2,7 +2,7 @@
 %global yum_install ((0%{?rhel} <= 7) && (0%{?rhel} >= 5)) || ((0%{?fedora} < 27) && (0%{?fedora} > 0))
 %global zypper_install (0%{?suse_version} > 0)
 
-%global build_tracer 0%{?rhel} >= 7 || 0%{?fedora}
+%global build_tracer (0%{?suse_version} > 0 ) || ((0%{?fedora}) || (0%{?rhel} >= 7))
```

That means we support Tracer on SUSE now?

---

Yes! It's not using the tracer library, though. It's much more basic: https://github.com/Katello/katello-host-tools/blob/master/src/zypper_plugins/tracer_upload.py

---

Nice.

---

Isn't the `> 0` redundant? I think logically the following is equivalent:
```suggestion
%global build_tracer 0%{?rhel} >= 7 || 0%{?fedora} || 0%{?suse_version}
```

---

While I preferred the inconsistent, confusing, and needlessly verbose solution I'll take your suggestion :smile: